release-tools/prow.sh

```diff
@@ -652,103 +515,74 @@ find_deployment () {
-  # Wait until volumesnapshot CRDs are in place.
-  CRD_BASE_DIR="https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/${CSI_SNAPSHOTTER_VERSION}/config/crd"
-  kubectl apply -f "${CRD_BASE_DIR}/snapshot.storage.k8s.io_volumesnapshotclasses.yaml" --validate=false
-  kubectl apply -f "${CRD_BASE_DIR}/snapshot.storage.k8s.io_volumesnapshots.yaml" --validate=false
-  kubectl apply -f "${CRD_BASE_DIR}/snapshot.storage.k8s.io_volumesnapshotcontents.yaml" --validate=false
+# Installs all nessesary CRDs  
+install_crds() {
+  # Wait until cosi CRDs are in place.
+#  CRD_BASE_DIR="https://github.com/kubernetes-sigs/container-object-storage-interface-api/crds"
+  CRD_BASE_DIR="https://raw.githubusercontent.com/brahmaroutu/api/proper_crds/crds"
+  kubectl apply -f "${CRD_BASE_DIR}/objectstorage.k8s.io_bucketclasses.yaml" --validate=false
+  kubectl apply -f "${CRD_BASE_DIR}/objectstorage.k8s.io_bucketrequests.yaml" --validate=false
+  kubectl apply -f "${CRD_BASE_DIR}/objectstorage.k8s.io_buckets.yaml" --validate=false
+  kubectl apply -f "${CRD_BASE_DIR}/objectstorage.k8s.io_bucketaccessclasses.yaml" --validate=false
+  kubectl apply -f "${CRD_BASE_DIR}/objectstorage.k8s.io_bucketaccessrequests.yaml" --validate=false
+  kubectl apply -f "${CRD_BASE_DIR}/objectstorage.k8s.io_bucketaccesses.yaml" --validate=false
   cnt=0
-  until kubectl get volumesnapshotclasses.snapshot.storage.k8s.io \
-    && kubectl get volumesnapshots.snapshot.storage.k8s.io \
-    && kubectl get volumesnapshotcontents.snapshot.storage.k8s.io; do
+  until kubectl get bucketaccessclasses.objectstorage.k8s.io \
+    && kubectl get bucketaccessrequests.objectstorage.k8s.io \
+    && kubectl get bucketaccesses.objectstorage.k8s.io \
+    && kubectl get bucketclasses.objectstorage.k8s.io	 \
+    && kubectl get bucketrequests.objectstorage.k8s.io  \
+    && kubectl get buckets.objectstorage.k8s.io; do
     if [ $cnt -gt 30 ]; then
-        echo >&2 "ERROR: snapshot CRDs not ready after over 1 min"
+        echo >&2 "ERROR: cosi CRDs not ready after over 1 min"
         exit 1
     fi
-    echo "$(date +%H:%M:%S)" "waiting for snapshot CRDs, attempt #$cnt"
+    echo "$(date +%H:%M:%S)" "waiting for cosi CRDs, attempt #$cnt"
 	cnt=$((cnt + 1))
     sleep 2
   done
 }
 
-# Install snapshot controller and associated RBAC, retrying until the pod is running.
-install_snapshot_controller() {
-  kubectl apply -f "https://raw.githubusercontent.com/kubernetes-csi/external-snapshotter/${CSI_SNAPSHOTTER_VERSION}/deploy/kubernetes/snapshot-controller/rbac-snapshot-controller.yaml"
+# Install controller and associated RBAC, retrying until the pod is running.
+install_controller() {
+  #kubectl apply -f "https://raw.githubusercontent.com/kubernetes-sigs/container-object-storage-interface-controller/${COSI_CONTROLLER_VERSION}/deploy/controller.yaml"
+  kubectl apply -f "https://raw.githubusercontent.com/container-object-storage-interface/cosi-controller-manager/master/samples/rbac.yaml"
```

Comment:
See my comments earlier about install location should be dynamic.

Reply:
Fixed